integration-cli/docker_cli_plugins_test.go

```diff
@@ -217,7 +217,7 @@ func (ps *DockerPluginSuite) TestPluginInstallImage(c *testing.T) {
 
 	out, _, err := dockerCmdWithError("plugin", "install", repoName)
 	assert.ErrorContains(c, err, "")
-	assert.Assert(c, strings.Contains(out, `Encountered remote "application/vnd.docker.container.image.v1+json"(image) when fetching`))
+	assert.Assert(c, strings.Contains(out, `Encountered remote "application/vnd.docker.container.image.v1+json"(image) when fetching`), out)
```

Comment:
`cmp.Contains` should take care of this.

```
docker_cli_plugins_test.go:221: assertion failed: string "Error response from daemon: application/vnd.docker.distribution.manifest.v1+prettyjws not supported\n" does not contain "Encountered remote \"application/vnd.docker.container.image.v1+json\"(image) when fetching"
```

Reply:
Ah, yes, I can replace those in this file.

Is the output it received here (`application/vnd.docker.distribution.manifest.v1+prettyjws not supported`) the expected output? (So should the test just be updated to match the other error?) or is there a bug at hand?

Reply:
🤷 This looks like a schema1 manifest?
Seems like the media type changed as is, and then we've also moved this to use containerd distribution instead of docker/distribution.

It probably doesn't make sense to even support schema1 manifests for plugins... but I guess we could tell it to convert for us.